src/model/User.model.js

```diff
@@ -24,11 +24,17 @@
  * @property {string} server - server URL
  */
 export default class User {
-  constructor (username, groups, created, admin, server) {
+  constructor (username, groups, created, admin, server, owner) {
+    // the authenticated user
+    // (full info only available when authenticated via the hub)
     this.username = username
     this.groups = groups
     this.created = created
     this.admin = admin
-    this.server = server
+    this.server = server || '?' // server can be unset
```

Comment:
Could we just leave it blank? Or is the `?` necessary or used elsewhere?

Reply:
I think I made this change because I was getting undefined errors from somewhere in the code.

Reply:
Quite sure this can be removed @oliver-sanders . Maybe it was the unit test that failed when it was undefined?

From what I recall, I used this `server` in the past to craft the GraphQL endpoint URL. But then we ran into problems after a user reported s/he couldn't get `/extra-path/users/blabla/#/` to work, and then I stopped using `server` and used the `window.location` to create the URL.

Only reason the value is still saved, I think, is because it's returned from the JupyterHub REST service, so no harm in leaving it.

Reply:
Ah that might have been it, will take a look...